content/1.7/back-office/catalog/discount/add-edit-cart-price-rule.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Adding and Editing cart rules
+weight: 1
+---
+
+# Cart Rule > Edit
+When you enter the Cart rule editing UI, there are 3 separate tabs - Information, Conditions and Actions. Each of them are combined together with the general cart rules functionality.
+
+The explanations of the input fields, dropdowns, switch toggle buttons and text fields are stated below:<br>
+
+## Information tab
+
+![Cart Rules: Information tab](/static/img/cart-rules-information-tab.png)
+
+- **Name** - it is a mandatory field, it can be set as unique rule name. There is a widget at the right of this field with multiple languages. The different languages can be set to the rule names for this certain Information tab form.
+- **Description** - it can be described where is the cart rule used for in more details.
+- **Code** - unique cart rule code can be set here, or simply by clicking Generate and Prestashop will generate the rule automatically with unique code name. It will become as voucher from front end side.
+- **Highlight switch toggle button** - this option can be enabled, then the cart rule voucher will be displayed in cart summary. When there is no Voucher Code set, the Highlight switch toggle button is hidden.
+- **Partial use switch toggle button** - this option can be enabled, when the total cart sum is smaller than the voucher sum. If Partial usage is disabled, voucher value will be lovered to the total order amount. If it will be enabled, new voucher will be created in the Cart, as a reminder. Partial use toggle button is switched on by default.
```

Comment:
```suggestion
- **Partial use switch toggle button** - If enabled, when the total cart of the checkout amount is smaller than the cart rule amout, a cart rule is created with the remaining sum.
```
What happens when the action is free shipping or percentage or amount and applied to the order or the specific product while use the partial use feature ?
Is the new cart rule has code? Does the condition match the previous one? 
Does it duplicate the quantity of available? Does the quantity of available is decresead?

Reply:
@MatShir hey, yes good point, you are right. I have tested, and it becomes a new cart rule with the new cart rule name with "-2" prefix with the remaining sum. The conditions are almost the same, except the cart rule quantity availability - it is just 1 for each user.